.gitignore

```diff
@@ -1,2 +1,3 @@
 *~
 build*
+/.vscode/
```

Comment:
Please put a newline at the end of the file.  It might be better to have your editor setup to do it for you.